miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSequencingParametersService.java

```diff
@@ -148,13 +149,25 @@ private void validateRunType(SequencingParameters params, List<ValidationError>
     }
   }
 
+  private void validateMovieTime(SequencingParameters params, List<ValidationError> errors) {
+    if(params.getInstrumentModel().getPlatformType() == PlatformType.PACBIO) {
+      if(params.getMovieTime() < 0) {
+        errors.add(new ValidationError("movieTime", "Movie time must be greater than 0"));
+      }
+    } else if (params.getMovieTime() != null) {
```

Comment:
* Check for null first. If movie time is less than zero, this will currently throw a `NullPointerException`.
* You're checking whether the value is less than zero, but the message says it must be greater than zero. Is zero allowed or not? I don't think it should be.